Portfolio headline with social media buttons and decorative elipces

Sirvelasque opened the pull request:
Add toolbar skeleton, my name and roll to start and go with supporting text which include skills, framework and known languages, to then finish with buttons linked to social media with the respecting icon for each ending with a button to the Works section

Sirvelasque committed:
Create linters.yml

css and html linters

Sirvelasque committed:
Create index.html

create html file and index document

Sirvelasque committed:
Create style.css

Sirvelasque committed:
add nav and images file

add html for nav bar with my logo text and bars button by fontawesome

Sirvelasque committed:
positioning logo and toolbar

also add the background color

Sirvelasque committed:
name and roll

add my profession and name, gave it style

Sirvelasque committed:
add support text

Sirvelasque committed:
background changes

Sirvelasque committed:
add list and positioning 1st divider

Sirvelasque committed:
finish suporting text and ading icons

Sirvelasque committed:
Positioning supproting items

social media buttons and slide down work button

Sirvelasque committed:
add top circles

add top elipses with blur filter

Sirvelasque committed:
stard adding info to README.md

Sirvelasque committed:
identation and empty lines correction

Sirvelasque committed:
Update style.css

Sirvelasque committed:
style.css linter errors correction, identation

Sirvelasque committed:
style errors fixed

Sirvelasque committed:
put alt to every img

medaminedev66 commented on html/index.html:
```diff
@@ -0,0 +1,80 @@
+
+    <div class="headline">
+      <div class="presentation">
+        <div class="my_name">Walter Velasquez</div>
```
It would be better to use `h1` tag for this main title instead of `div` tag to improve our structure and semantics.

Hamzaoutdoors replied:
- [x] Issue well resolved ✔️

medaminedev66 commented on html/index.html:
```diff
@@ -0,0 +1,80 @@
+    <div class="headline">
+      <div class="presentation">
+        <div class="my_name">Walter Velasquez</div>
+        <div class="roll">Software developer</div>
```
We can use `h2` tag for this subtitle instead of `div` tag to improve our structure and semantics.

Hamzaoutdoors replied:
- [x] Issue well resolved ✔️

medaminedev66 commented on html/index.html:
```diff
@@ -0,0 +1,80 @@
+      </nav>
+    </header>
+
+    <div class="headline">
```
You have done a great job with implementing the headline section. Let's make your code better by using the `section` tag for the headline section instead of the `div` tag to improve our structure and semantics.

Hamzaoutdoors replied:
- [x] Issue well resolved ✔️

medaminedev66 commented on css/style.css:
```diff
@@ -0,0 +1,195 @@
+  align-self: center;
+}
+
+.work-btn a {
+  width: 47px;
+  height: 32px;
+  top: 4px;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #a2a3ae;
+  text-decoration: none;
+}
+
+.top_circles {
+  position: absolute;
+  width: 513px;
+  height: 563px;
+  left: 5px;
+  top: 97px;
+}
+
+.elipse_1 {
+  position: absolute;
+  width: 160px;
+  height: 160px;
+  left: -106px;
+  filter: blur(50px);
+  background-color: #edecff;
+}
+
+.elipse_2 {
+  position: absolute;
+  width: 68px;
+  height: 68px;
+  left: 336px;
+  top: 495px;
+  filter: blur(50px);
+  background-color: #edecff;
+}
```
- [ ] Please also as required in the project requirements, you must stick to the design as much as possible using the Figma Template provided. I strongly recommend using the fonts that are provided in the Figma design.
- [ ] Please make the headline section responsive as much as you can, I would suggest making the width 100% so your width is not fixed and can fit different mobile screen sizes.
Your Implementation | Figma Design
:---:|:---:
<img width="136" alt="image" src="https://user-images.githubusercontent.com/77190203/149217049-40788d9c-8c9a-4928-b01b-ea050abb4c8f.png"> | <img width="123" alt="image" src="https://user-images.githubusercontent.com/77190203/149216890-b917f5dd-bba3-43b9-b65d-ba6de34b1776.png">

Hamzaoutdoors replied:
- [x] Issue well resolved ✔️

medaminedev66 reviewed (commented):
Hi @Sirvelasque , :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- ✔️Git/GitHub flow best practices have been followed
- ✔️There are no linter errors
- ✔️You have a descriptive PR

## Required Changes ♻️

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/V

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Sirvelasque committed:
font change and semantic html improve

change the font from sans to Roboto
change div tag on headline for section
change titles tags from div to h1 and h2

Hamzaoutdoors commented on html/index.html:
```diff
@@ -0,0 +1,83 @@
+      </div>
+
+      <div class="sup_items">
+        <div class="social_media">
+          <div class="social_icon"><a href="#"><img alt="My Github" src="../img/icons/icongit.svg"></a></div>
+          <div class="social_icon"><a href="#"><img alt="My in icon" src="../img/icons/iconin.svg"></a></div>
+          <div class="social_icon"><a href="#"><img alt="My angelist" src="../img/icons/iconangel.svg"></a></div>
+          <div class="social_icon"><a href="#"><img alt="My twitter" src="../img/icons/icontw.png"></a></div>
+          <div class="social_icon"><a href="#"><img alt="My Medium" src="../img/icons/iconm.png"></a></div>
+        </div>
```
- [Required] :
Please make sure that the social media icons ("GitHub", "Twitter") are in a `ul` tag, each of them in a `li `tag. ; each icon has a link (`a` tag)

Sirvelasque replied:
so the only thing requiered is to changes the div tags, ul for the container and li for each item? and just to be sure, the a tag should cover the whole li(a-li -img-</></>) , or is it fine for it to just cover the icon( li -a-img</></>)?

Hamzaoutdoors reviewed (changes requested):
Hey @Sirvelasque, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![image](https://media.giphy.com/media/KyBesf8kNT6s88SEgb/giphy.gif)

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README

## Required Changes ♻️

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**N.A**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Sirvelasque committed:
change social media tags from div to ul and li

change the container from a div to ul and the icons are now list items, also linked the icons to my accounts

Sirvelasque committed:
identation fix line 150

julie-ify reviewed (approved):
## Hi @Sirvelasque,

# Highlights
- Good job implementing the required changes from the previous reviewers ✔️ 

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

![Approved emoji](https://user-images.githubusercontent.com/81468675/147213535-ab80a883-5bf8-4284-b9ee-d90332328671.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
